scrapers/covid-19/govScrapers/getIsrael.js

```diff
@@ -21,30 +22,44 @@ const params = {
  */
 const parseData = (data) => {
 	const dateRegex = /\d+-\d+-\d+/g;
-	const patientsPerDay = data[2].data;
-	const deadPatientsPerDay = data[3].data;
-	const recoveredPerDay = data[4].data.splice(20, data[4].data.length - 1);
-	const testsPerDay = data[5].data.splice(30, data[5].data.length - 1);
+	const patientsPerDay = data[1].data;
+	const deadPatientsPerDay = data[2].data;
+	const recoveredPerDay = data[3].data.splice(20, data[3].data.length - 1);
+	const testsPerDay = data[4].data.splice(30, data[4].data.length - 1);
+	const doctorData = data[6].data[0];
 	const timeline = patientsPerDay.map((elem, index) => ({
 		date: elem.date.match(dateRegex)[0],
 		newHospitalized: elem.new_hospitalized,
-		totalhospitalized: elem.Counthospitalized,
+		totalHospitalized: elem.Counthospitalized,
+		homePatients: elem.patients_home,
+		hotelPatients: elem.patients_hotel,
 		totalBeds: elem.total_beds,
-		StandardOccupancy: elem.StandardOccupancy,
+		standardOccupancy: elem.StandardOccupancy,
 		newDeaths: deadPatientsPerDay[index].amount,
 		newlyRecovered: recoveredPerDay[index].amount,
 		newTestsTaken: testsPerDay[index].amount,
-		newPositiveTests: testsPerDay[index].positiveAmount
+		newPositiveTests: testsPerDay[index].positiveAmount,
+		activeNoncritical: elem.CountEasyStatus,
+		activeModerate: elem.CountMediumStatus,
+		activeCritical: elem.CountHardStatus,
+		onVentilators: elem.CountBreath
 	}));
 	return {
 		updated: new Date(data[0].data.lastUpdate).valueOf(),
 		data: {
-			sickLocated: { home: data[1].data[0].amount, hospital: data[1].data[1].amount },
-			sickByAge: data[6].data.map((entry) => {
+			sickByAge: data[5].data.map((entry) => {
 				const { section, male, female } = entry;
 				return { section: `ages ${section} - ${section + 10}`, male, female };
 			}),
+			doctorData: {
```

Comment:
I think it should be called healthPersonnelData or healthStaffData as it includes not just doctors, And that is what its labeled on the gov website.
![Annotation 2020-07-06 105403](https://user-images.githubusercontent.com/12494197/86569629-5f5a7a80-bf77-11ea-9ce1-f10274496915.png)